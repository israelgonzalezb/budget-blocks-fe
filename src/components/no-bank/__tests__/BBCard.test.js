
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import BBCard from '../BBCard';


describe('BBCard test', () => {
  it('BBCard should match snapshot', () => {
    const component = renderer.create(<BBCard
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // hey there...
  });
});
