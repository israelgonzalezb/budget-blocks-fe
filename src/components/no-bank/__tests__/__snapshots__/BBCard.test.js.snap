// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BBCard test BBCard should match snapshot 1`] = `
<div
  className="MuiPaper-root MuiCard-root makeStyles-default-2 MuiPaper-outlined MuiPaper-rounded"
>
  <div
    className="MuiCardContent-root"
  >
    <h2
      className="MuiTypography-root MuiTypography-h5"
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    />
    <p
      className="MuiTypography-root MuiTypography-body1"
    />
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    />
  </div>
</div>
`;
